geeksay.js

```diff
@@ -104,6 +104,7 @@ const translations = {
     "heart": "<3",
     "love": "<3",
     "like": "<3",
+    "at": "@"
```

Comment:
This translation already exists on line 64:
https://github.com/swapagarwal/geeksay/blob/8d0d99e23cb6d772d66c85cf0650954f96758822/geeksay.js#L64